pages/linux/ufw.md

```diff
@@ -4,33 +4,33 @@
 > Frontend for `iptables` aiming to make configuration of a firewall easier.
 > More information: <https://wiki.ubuntu.com/UncomplicatedFirewall>.
 
-- Enable ufw:
+- Enable `ufw`:
 
 `ufw enable`
 
-- Disable ufw:
+- Disable `ufw`:
 
 `ufw disable`
 
-- Show ufw rules, along with their numbers:
+- Show `ufw` rules, along with their numbers:
 
 `ufw status numbered`
 
 - Allow incoming traffic on port 5432 on this host with a comment identifying the service:
 
-`ufw allow {{5432}} comment "{{Service}}"`
+`ufw allow 5432 comment "{{service}}"`
```

Comment:
This is a user added comment. It would be natural for the comment to start with a capital letter.